eZ/Bundle/EzPublishCoreBundle/DependencyInjection/Configuration/Parser/Content.php

```diff
@@ -34,9 +34,13 @@ public function addSemanticConfig(NodeBuilder $nodeBuilder)
                     ->arrayNode('tree_root')
                         ->canBeUnset()
                         ->children()
+                            ->scalarNode('location_remote_id')
+                                ->info("Root locationRemoteId for routing and link generation.\ If set, it will be used instead of location_id. ")
+                                ->cannotBeEmpty()
+                            ->end()
                             ->integerNode('location_id')
-                                ->info("Root locationId for routing and link generation.\nUseful for multisite apps with one repository.")
-                                ->isRequired()
+                                ->info("Root locationId for routing and link generation.\nUseful for multisite apps with one repository. It won't be used if location_remote_id is set")
+                                ->cannotBeEmpty()
```

Comment:
can we add a constraint that both this node and the one above can not be set at the same time ?

Reply:
ok. sounds good.